src/main/java/com/hackerrank/algorithms/test/TestTwoString.Java

```diff
@@ -0,0 +1,52 @@
+package com.hackerrank.algorithms.test;
```

Comment:
Some review comments:
- The file location isn't correct. See file structure for maven projects and see where "tests" classes should reside.
- File is not formatted properly. Default IntelliJ formatting would suffice here.